### PR TITLE
feat/make it json

### PR DESCRIPTION
**Pull request by mattiasliljenzin**
Use the same format we are using with Serilog

### CODE REVIEW

File: src/structuredLogger.js [hunk 1]
```diff
@@ -10,69 +10,56 @@ function write(logData) {
...
-  const logObject = {
-    Stack: stack
-  };
-
-  if (!metadata) {
-    return logObject;
-  }
+function createlogData(message, args, logLevel = 'INFO') {
+  const safeArgs = args instanceof Object ? args : {};
+  const metadata = Object.assign(safeArgs || {}, baseMetadata);
```
Comment (steve-mathem) [inline src/structuredLogger.js:15]: seems like `safeArgs` will always be valid, so how about:
```
const metadata = {
   ...baseMetadata,
   ...safeArgs
};

Comment (mattiasliljenzin) [reply src/structuredLogger.js:15]: Even if args is a simple string?

Comment (steve-mathem) [reply src/structuredLogger.js:15]: well, a string is not `instanceof Object` (unless you do `new String('')` maybe)
 so `safeArgs` is either `args` when it's an object, or it's `{ }`, so I'm pretty sure the right hand side of `||` will never run

Comment (mattiasliljenzin) [reply src/structuredLogger.js:15]: ah, good point. feel free to adjust it :)

**Pull request merged**